src/NumericalAlgorithms/LinearOperators/PartialDerivatives.tpp

```diff
@@ -200,27 +200,12 @@ constexpr tnsr::iab<double, 3> Killing_vector_derivatives() {
   // holds derivative of both (0,-y,x,0) for \partial_y and (0,-z,0,x) for
   // \partial_z
   tnsr::iab<double, 3> da_killing_vectors{};
```

Comment:
I don't think this guarantees the entries are zeroed.  Use `da_killing_vectors{0.0}` to ensure that.

Reply:
Is there a reason it needs to be constexpr?

Reply:
For some reason I thought this was in a test file.  Looks good, then.